README.md

```diff
@@ -58,7 +58,9 @@ Usage of ./observatorium:
   -debug.name string
     	A name to add as a prefix to log lines. (default "observatorium")
   -internal.tracing.endpoint string
-    	The full URL of the trace collector. If it's not set, tracing will be disabled.
+    	The full URL of the trace agent or collector. If it's not set, tracing will be disabled.
+  -internal.tracing.endpoint-type string
```

Comment:
Just curious why do we need this? What does it change?

Reply:
Agents and collector uses different protocols (IIRC Agent uses Thrift over UDP and collector uses HTTP).